api/catalog/api/models/media.py

```diff
@@ -135,7 +135,7 @@ def __str__(self):
         return f"{self.__class__.__name__}: {self.identifier}"
 
 
-class AbstractMediaReport(OpenLedgerModel):
+class AbstractMediaReport(models.Model):
```

Comment:
If we wanted to share the same base class, it is possible to tell Django to map a column to a different column name than the attribute name. We'd just need to overwrite the attribute in the sub-classing model. Totally up to you though, this approach also works fine.

Reply:
It definitely is, but it would complicate the PR even more, mix a bunch of different concerns in a migration and leave the PR non-atomic. We can circle back to it later.

Reply:
> mix a bunch of different concerns in a migration and leave the PR non-atomic

In what sense would it make the PR non-atomic? My understanding is that it would allow us to re-use the existing column with a unified API on the Python side without needing to worry about the column names being slightly different (which the code _would_ need to care about when we're referencing the columns, but not if we did the aliasing I suggested).

I agree it can be done later, but I don't understand how this would affect the atomic nature of the PR. At the very least it might cut down on extraneous, repeated documentation, especially if the column name change was abstracted out into a separate mixin or something to be used by the cases that need it.

I do think it should be done eventually though, if only to unify the attribute names on the code side so we don't have to try to remember (or be frustrated/confused when we forget) whether one model or another uses `created_on` or `created_at`.

Reply:
I meant that the PR was working on reducing DB queries so fixing timestamp column names was not directly related to the cause of the PR but rather a drive-by refactor.

Reply:
Ohhhh, sorry! Sure thing. The PR felt like it was touching lots of things so it was hard to tell what was in scope, but understood :+1: I misunderstood "atomic" to mean what it means with databases :facepalm: Apologies.